src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Rows.component.tsx

```diff
@@ -37,20 +44,35 @@ export function TableContentRowsComponent(props: {
         </TableCell>
     );
     const k = Object.keys(props.row);
-    const colKeys = _.pull(k, "name")
-    colKeys.forEach((element): void => {
+    const colKeys = _.pull(k, "name");
+    let rowSum = 0;
+    colKeys.forEach((colKey): void => {
+        const cellNumber = props.row[colKey]
+        rowSum += Number.parseFloat(cellNumber);

```

Comment:
See above: parseInteger, i think.

Reply:
I wonder why you check against undefined for the col sums but not her for the row sum?

Reply:
Done. Commit "Change parseFloat() to parseInt()"

Reply:
Undid it. I need parseFloat to display the relative values correctly.